src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,15 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[3];
+        machines[0] = new Truck();
+        machines[1] = new Bulldozer();
+        machines[2] = new Excavator();
+        for (Machine mash : machines) {
```

Comment:
let's use full words for variables
```suggestion
        for (Machine machine : machines) {
```